src/yunohost/app.py

```diff
@@ -2593,6 +2593,29 @@ def _raise_invalid_answer(self, question):
                             error=m18n.n('user_unknown', user=question.value))
 
 
+class NumberArgumentParser(YunoHostArgumentFormatParser):
+    argument_type = "number"
+    default_value = ""
+
+    def parse_question(self, question, user_answers):
+        question = super(NumberArgumentParser, self).parse_question(question, user_answers)
+
+        if question.default is None:
+            question.default = 0
+
+        return question
+
+    def _post_parse_value(self, question):
+        if isinstance(question.value, int):
+            return super(NumberArgumentParser, self)._post_parse_value(question)
+
+        try:
+            return int(question.value)
```

Comment:
you can use `str.isdigit()` instead here if you want

Reply:
Thank you!